editor/components/toolbar/index.js

```diff
@@ -28,7 +28,7 @@ function Toolbar( { controls } ) {
 					} }
 					className={ classNames( 'editor-toolbar__control', {
 						'is-active': control.isActive
-					} ) } />
+					}, control.classNames ) } />
```

Comment:
The change that used the classname got backed out, it was created to pass a class to the control icon and provide a different color. It could be useful to leave for others to be able to tweak controls.